app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from __future__ import annotations
+from typing import Optional
+
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: Optional[list[Player]]) -> int:
```

Comment:
`players `argument is not optional, it is required. Remove `Optional `typing here.